lib/composer.js

```diff
@@ -590,5 +598,6 @@ function createDefaultAdapter (name, keyFields) {
     return result
   }
 }
+function noop () {}
```

Comment:
Do we really want the default behavior to be swallowing errors? That seems like it will lead to hard to debug issues for users.

Reply:
I agree, this does not seem like a good idea

Reply:
should `onSubgraphError` be required?